artifactory/commands/buildinfo/publish.go

```diff
@@ -248,3 +275,187 @@ type partialModule struct {
 	artifacts    map[string]buildinfo.Artifact
 	dependencies map[string]buildinfo.Dependency
 }
+
+// Set build property for each dependency.
+func addDependenciesProps(partialBuildsInfo []*buildinfo.PartialBuildInfo, sm *artifactory.ArtifactoryServicesManager, threads int) error {
+	// Key: dependency sha1, Value: pointer to dependency build property.
```

Comment:
Should the value description be - 
...a **list** of pointers to dependency build property.